lang/extract_json_strings.py

```diff
@@ -925,6 +925,17 @@ def extract_snippets(item):
             writestr(outfile, snip["text"])
 
 
+def extract_vehicle_part_category(item):
+    outfile = get_outfile("vehicle_part_categories")
+    name = item.get("name")
+    short_name = item.get("short_name")
+    comment = item.get("//")
+    short_comment = "(short name, optimal 1 symbol) " + comment
+    writestr(outfile, name, context="vpart_category_name", comment=comment)
+    writestr(outfile, short_name, context="vpart_category_short_name",
+             comment=short_comment)
```

Comment:
I'm a bit behind on CDDA notifications, but putting the categories here won't work; they need to go into the JSON so that they match the categories in the `translation` object, otherwise the arguments to `pgettext` won't find the right string to translate.

Reply:
I'm not sure I got this right, the categories in JSON are parsed by extract_json_strings.py
I can see if i run the py script it generates a new file inside https://github.com/CleverRaven/Cataclysm-DDA/tree/master/lang/json
And there's a readme in there that says those files shouldn't be committed

Can you expand where I need to put changes or point to a commit that does it right?

Reply:
Fixed in #46799.